style.css

```diff
@@ -0,0 +1,110 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  background-color: #3e3c3c;
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.name2 {
+  display: flex;
+}
+
+body {
+  background-color: #1c1a19;
+}
+
+h1 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 50px;
+  line-height: 4rem;
+  color: #f55800;
+  width: 400px;
```

Comment:
- [ ] According to the project requirements, You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using the templates in Figma. Kindly make sure the `width` of the headline section (right after the header): of the portfolio matches the original Figma design Check the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). You have to use the same margins and width ✍🏻 

- Follow the original vision if it's pretty and works, it will be a wonderful website.. 🎨 🖌️ 

- Your Design:
![19](https://user-images.githubusercontent.com/79058364/185610487-fefb349e-3dfa-440d-9c60-d2c5a9153c30.png)



- Figma Design:

![image](https://user-images.githubusercontent.com/79058364/185610584-4d333dd7-4a38-4f40-adba-6678e82cf761.png)





- NOTE: Minimum mobile screen size should be `375`px. Otherwise, the mobile version should be responsive until the desktop screen size.